### ISSUE TITLE
Without autocomplete and does not share

### ISSUE THREAD

**Issue by claudioguolo** [Zimbra-Community/shared-mailbox-toolkit#49]
It is a very useful tool in my environment, but there have been problems for some admin accounts for a while.
It happens that the user panel is displayed in the webadmin, however the autocomplete does not work, the message remains in Loading Autocompletation forever, and even adding the email addresses manually and clicking the share button the progress message stays forever in Creating Share.
In the beginning there were some admin accounts that presented this, but now they are in all.
I have already restarted the services and the same situation remains.
Any logs that I can check or indication of some technique to solve this?

Zimbra 8.8.15-3888 NETWORK

Claudio

**Comment by barrydegraaff** [comment]
Try and run the command line version and see if that works, the UI is a wrapper around the command line. 

/usr/local/sbin/subzim 

Kind regards, 

Barry de Graaff 
Zeta Alliance 
Co-founder & Developer 
zetalliance.org | github.com/Zimbra-Community 

Signal: +31 617 220 227 
Fingerprint: 97f4694a1d9aedad012533db725ddd156d36a2d0

**Comment by claudioguolo** [comment]
[zimbra@server ~]$ subzim user@tjmt.jus.br dept@tjmt.jus.br
Setting the default permissions: rwixd
mailbox: dept@tjmt.jus.br, size: 161.41 MB, messages: 308, unread: 157
authenticated as dept@tjmt.jus.br
mbox dept@tjmt.jus.br>
mbox dept@tjmt.jus.br> mfg / account user@tjmt.jus.br rwixd
mbox dept@tjmt.jus.br> mfg /Inbox account user@tjmt.jus.br rwixd
ERROR: mail.GRANT_EXISTS (grantee already exists af31428b-9998-491a-8548-fb2ea58e95eb)
mbox dept@tjmt.jus.br> mfg /Sent account user@tjmt.jus.br rwixd
ERROR: mail.GRANT_EXISTS (grantee already exists af31428b-9998-491a-8548-fb2ea58e95eb)
mbox dept@tjmt.jus.br> mfg /Junk account user@tjmt.jus.br rwixd
ERROR: mail.GRANT_EXISTS (grantee already exists af31428b-9998-491a-8548-fb2ea58e95eb)
mbox dept@tjmt.jus.br> mailbox: user@tjmt.jus.br, size: 2.17 GB, messages: 3192, unread: 3
authenticated as user@tjmt.jus.br
mbox user@tjmt.jus.br>
mbox user@tjmt.jus.br> cm /dept@tjmt.jus.br dept@tjmt.jus.br /
10490
mbox user@tjmt.jus.br> dofrl "dept.persona.sent"
ERROR: zclient.CLIENT_ERROR (can't find rule: [Ljava.lang.String;@79df80a4)
mbox user@tjmt.jus.br> aofrl "dept.persona.sent" active any address "from" all contains "dept@tjmt.jus.br" fileinto "dept@tjmt.jus.br/Sent" stop
mbox user@tjmt.jus.br> prov>
prov> grr account dept@tjmt.jus.br usr user@tjmt.jus.br sendAs
prov> deleteIdentity user@tjmt.jus.br dept.persona
ERROR: service.PERM_DENIED (permission denied: can not modify options)
prov> createIdentity user@tjmt.jus.br dept.persona zimbraPrefFromAddress dept@tjmt.jus.br zimbraPrefWhenSentToAddresses dept@tjmt.jus.br zimbraPrefWhenSentToEnabled TRUE zimbraPrefFromDisplay "Varzea Grande - Forum - Gabinete 2 Vara Fazenda"
ERROR: service.PERM_DENIED (permission denied: can not modify options)
prov> [zimbra@server ~]$

**Comment by claudioguolo** [comment]
Hello, by the subzim command the sharing occurs normally.
Webadmin alone does not work properly.

Thanks for the help again.

Claudio

**Comment by barrydegraaff** [comment]
Can you 

      wget https://raw.githubusercontent.com/Zimbra-Community/shared-mailbox-toolkit/master/bin/acctalias -O /usr/local/sbin/acctalias

And then reload the admin ui and see if that fixes the auto completion. Then we can continue and see if the sharing works again from the UI.

**Comment by claudioguolo** [comment]
olá. Mesmo realizado o procedimento a mensagem "Loading auto completion..." continua sendo exibida permanentemente. 
thanks

**Comment by barrydegraaff** [comment]
I tested with your domain names, and it seems a recent change in the auto completion triggers "Too many ADMIN sessions". This will result in the UI not auto completing and fail to create shares.

https://wiki.zimbra.com/wiki/Steps_to_fix_Too_many_ADMIN_sessions_error

Check your lo (tail  -f /opt/zimbra/log/*) look for "Too many ADMIN sessions" while trying to create the share. If you see that message consider:

      zmlocalconfig -e zimbra_session_limit_admin=10
      zmmailboxdctl restart

Added a check in the installer for this issue.
